go.mod

```diff
@@ -127,3 +128,5 @@ require (
 	gorm.io/driver/sqlite v1.4.4 // indirect
 	gorm.io/gorm v1.25.7 // indirect
 )
+
+replace github.com/cloudfoundry/cloud-service-broker => github.com/cloudfoundry/cloud-service-broker v0.22.3-0.20240315140147-365141d530be
```

Comment:
I guess you need this for the temp pipeline - just a reminder to remove it before merging

Reply:
we actually pick releases for the CSB, so can't really do anything permanent until we release a new CSB version, hopefully today. That's the reason this PR is in draft mode still.

Reply:
updated